docs/src/main/asciidoc/security-oidc-auth0-tutorial.adoc

```diff
@@ -0,0 +1,743 @@
+[id="security-oidc-auth0-tutorial"]
+= Protect Quarkus web application by using `Auth0` OpenID Connect provider
+include::_attributes.adoc[]
+:diataxis-type: tutorial
+:categories: security,web
+
+xref:security-architecture.adoc[Quarkus Security] provides comprehensive OpenId Connect (OIDC) and OAuth2 support with its `quarkus-oidc` extension, supporting both xref:security-oidc-code-flow-authentication.adoc[Authorization code flow] and xref:security-oidc-bearer-token-authentication.adoc[Bearer token] authentication mechanisms.
+
+With Quarkus, you can easily configure OIDC providers such as link:https://www.keycloak.org/documentation[Keycloak], link:https://developer.okta.com/[Okta], link:https://auth0.com/docs/[Auth0], and other xref:security-openid-connect-providers.adoc[well-known social OIDC and OAuth2 providers].
+
+Learn how to use the Quarkus OpenID Connect extension (`quarkus-oidc`) together with https://auth0.com/docs/[Auth0] OIDC provider to protect your API endpoints.
+
+== Create Auth0 application
+
+Start with creating an `Auth0` application in the `Auth0` dashboard, for example, lets create a `QuarkusAuth0` web application:
```

Comment:
```suggestion
Start with creating an `Auth0` application in the `Auth0` dashboard, for example, let's create a `QuarkusAuth0` web application:
```
But if following the Quarkus template numbered steps approach:

```suggestion
. Go to the `Auth0` dashboard and create a regular web application. For example, create an `Auth0` application called `QuarkusAuth0`.
```

@sberyozkin - Should I pause suggesting edits to convert the actual steps into numbered sub-steps in this review iteration?   Thanks.

Reply:
@michelle-purcell I'm applying numbering scheme suggestions, but I'm not sure I'm following where to put `. ` next :-), please propose a few suggestions :-)